### PR TITLE
[MRG] CS328 Submission: Group <6>

### PR DESCRIPTION
**Pull request by rohithkumar27**
#### Lectures

- `<Date 1 03/02/22/Thursday>` : `<Greedy_algorithm_for_Densest_subgraphs>` : `<2022_02_03_Greedy_algorithm_for_Densest_subgraphs.ipynb>`
- `<Date 2 10/02/22/Thursday>` : `<Expansion_Conductance_Graphs_Sparset_Cut>` : `<2022_02_10_Expansion_Conductance_Graphs_Sparset_Cut.ipynb>`

#### Group Seq Number

`<6>`

#### Assigned Deadline

`<23/10/22/wednesday>`

#### Members
- `<Rohith Kumar>` : `< 19110103>` : `<rohith.tkr@iitgn.ac.in>` : `< rohithkumar27>`
- `<G.B Harshavardhan>` : `<19110083>` : `<gbh.vardhan@iitgn.ac.in>` : `<harsha1218>`
- `<Rithik Maligi>` : `<19110098>` : `<rithik.maligi@iitgn.ac.in>` : `<rithikmaligi>`

#### Any other comments?
The deadline was extended for one day.

### CODE REVIEW

**Comment by yadav-sachin** [comment]
@rohithkumar27 
Before I merge, 
1. Fix the pre-commit error 
       It's due to a missing newline at end. To solve, Run pre-commit --> add the changes --> commit & push the changes.
2.  Some images are not loading (maybe due to wrong path in markdown).
    https://153-443822015-gh.circle-artifacts.com/0/CS328-Notes/notebooks/2022_02_10_Expansion_Conductance_Graphs_Sparset_Cut.html#k-clique-combination

**Comment by rohithkumar27** [comment]
@yadav-sachin 
The images are loading in the local server after the build. I couldn't understand why they are not appearing in the link you shared.

File: CS328-Notes/notebooks/2022_02_10_Expansion_Conductance_Graphs_Sparset_Cut.ipynb [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+    "\n",
+    "## Sparsest Cut\n",
+    "\n",
+    "For a graph G ,Cut resulting the subset with minimum Conductance$(S,\\overline{S})$\n",
+    "\n",
+    "* It is also the conductance of Graph G(V,E).\n",
+    "\n",
+    " $$ Conductance(G)=Min(Conductance(S\\subset G(V,E))) $$\n",
+    "\n",
+    "```{figure} ../assets/2022_02_10_Expansion_conductance_graph_sparsecut/sparse_cut.jpeg\n",
```
Comment (yadav-sachin) [inline CS328-Notes/notebooks/2022_02_10_Expansion_Conductance_Graphs_Sparset_Cut.ipynb:159]: `../assets/2022_02_10_Expansion_conductance_graph_sparsecut/sparse_cut.jpeg\n` needs to be in next line after ```{figure}

File: CS328-Notes/_toc.yml [hunk 2]
```diff
@@ -24,3 +24,5 @@ parts:
   - file: notebooks/2022_01_27_Hierarchial_Clustering.md
   - file: notebooks/2022_01_31_HierarchialClustering_LinkagebasedClusturing.ipynb
   - file: notebooks/2022_02_01_DensestSubgraph_Maxflowalgorithm.ipynb
+  - file: notebooks/2022_02_03_Greedy_algorithm_for_Densest_subgraphs.ipynb
+  - file: notebooks/2022_02_10_Expansion_Conductance_Graphs_Sparset_Cut.ipynb
```
Comment (yadav-sachin) [inline CS328-Notes/_toc.yml]: give a newline at the end 🙄 
the pre-commit would pass then

**Comment by yadav-sachin** [comment]
@rohithkumar27 
Please commit the required changes by 25 March 11:59 PM.
- resolving pre-commit fail
- images load error

File: CS328-Notes/notebooks/2022_02_10_Expansion_Conductance_Graphs_Sparset_Cut.ipynb (hunk 1 above)
Comment (rohithkumar27) [reply CS328-Notes/notebooks/2022_02_10_Expansion_Conductance_Graphs_Sparset_Cut.ipynb:159]: I also looked into other codes about this image loading issue, but I haven't seen any difference. They used ../assets ... in the same line after ```{figure }.

**Comment by yadav-sachin** [comment]
@rohithkumar27 
I have corrected the mistakes. There will be some penalty for them.

**Pull request merged**